etc/local-testnet.sh

```diff
@@ -102,6 +102,7 @@ else
         target/${RELEASE}/telcoin-network genesis \
             --datadir "${ROOTDIR}" \
             --chain-id 0x1e7 \
+            --epoch-duration-in-secs 60 \
```

Comment:
60 second epochs for testnet seem OK?  I think we should shorten these for testing but maybe this is too far?